Lots of fixes and additions

ejolly opened the pull request.

ljchang commented on nltools/stats.py:
```diff
@@ -1191,3 +1192,175 @@ def procrustes(data1, data2):
+        mat1 = np.pad(mat1,((0,0),(0, mat2.shape[1] - mat1.shape[1])),'constant')
+    
+     _,_,sse = procrustes(mat1,mat2)
+    sse = 1 - sse # flip to similarity measure
```
are you sure that sum of squared error is scaled between 0 and 1?  This seems strange.  I think it's okay to leave it in distance.  There is a function in adjacency to approximate similarity from distance and the reverse (works for euclidean distance too)

ejolly replied:
I thought so based on the literature but it looks like scipy’s implementation might not be. Basically need to divide sse outputted here by sse of centered X matrix so it becomes a ratio. Thats the matlab approach and approach in the literature. Ill update and check it.

ejolly commented:
@ljchang I just need to fix procrustes distance so that it's standardized between 0-1 and then this PR should be ready to merge.

ejolly commented:
Full tally of this PR so far:
1. ADDED Distance correlation
2. ADDED (NEEDS FIX) Procrustes distance
3. BUG FIX `holm_bonf` acts like `fdr`
4. BUG FIX `fdr` handles edge-cases
5. BUG FIX/ENHANCEMENT `check_numpy_square_matrix` does consistent validation and conversion
5. ENHANCEMENT `expand_mask`, `collapse_mask` take custom masks 
6. ENHANCEMENT `Brain_Data.standarize()` takes axis argument and swapped for sklearn scaler

ejolly commented:
@ljchang This should be ready to go. Haven't gotten around the fixing the procrustes distance, but for now it just raised a `NotImplementedError`.

ejolly commented:
Ok I also just did an overhaul of *all* our files to make them PEP8 compliant and removed a bunch of unused imports in various files. Would recommend everyone pull in these changes on their local computers before adding new changes so we don't mess up formatting.

ljchang commented on nltools/utils.py:
```diff
@@ -218,10 +229,14 @@ def check_square_numpy_matrix(data):
 
-    if data.shape[0] != data.shape[1]:
-        raise ValueError('Make sure data is square.')
+    if len(data.shape) != 2:
```
You now have a silent error if data.shape isn't len 2.  This could be problematic when the whole point of the function is to check if it's square.  Can you fix this, or revert back?  I don't understand why you made this change.

ljchang commented:
Having a hard time reviewing all of the code.  Skimmed through most of it.  Hard to tell, which are bug fixes and which are formatting errors.  Going to go ahead and merge.  Let's keep an eye on any weird behaviors that emerge.

Nice work @ejolly!
